packages/x-date-pickers/src/MultiSectionDigitalClock/MultiSectionDigitalClock.tsx

```diff
@@ -397,11 +397,11 @@ export const MultiSectionDigitalClock = React.forwardRef(function MultiSectionDi
       return views;
     }
     const digitViews = views.filter((v) => v !== 'meridiem');
-    const result: TimeViewWithMeridiem[] = digitViews.toReversed();
+    digitViews.reverse();
```

Comment:
`.toReversed` is too recent, I avoid using it.
It was causing the following issue in the CI when building the doc:

```
9:25:02 AM: TypeError: ec.filter(...).toReversed is not a function
```

Reply:
Circle CI is on Node 18, `toReversed` is Node 20.